srcpkgs/wally-cli/files/50-wally.rules

```diff
@@ -0,0 +1,8 @@
+# Teensy rules for the Ergodox EZ
```

Comment:
did you come up with them?

Reply:
I got them from https://github.com/zsa/wally/wiki/Linux-install#2-create-a-udev-rule-file

I've written this in the file at `files/50-wally.rules`